README.md

```diff
@@ -28,16 +28,6 @@ fn main() {
 }
 ```
 
-# Tests
```

Comment:
This info is now moved into the tests.sh script, so maybe refer to that here in the README.

Reply:
Added note about the `tests.sh` script  :-)